### PR TITLE
docs: Add the USE_IMAGE_DEFAULT_PORTS word

### PR DESCRIPTION
**Pull request by harrisonliu5**
### What type of PR is this?

/kind documentation

### Which issue(s) this PR fixes:

Fixes #

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: locales/en/l10n-multiclusterProjects-applicationWorkloads-deployments-list.js [hunk 1]
```diff
@@ -46,19 +49,23 @@ module.exports = {
...
   IMAGE_PLACEHOLDER: 'Image name or path such as nginx:latest',
   IMAGE_EMPTY: 'Please set an image.',
   ENTER_POSITIVE_INTEGER_DESC: 'Please enter a positive integer.',
-  TOTAL_REPLICAS_EMPTY_DESC: 'Please enter the total number of pod replicas in all clusters.',
+  TOTAL_REPLICAS_EMPTY_DESC:
+    'Please enter the total number of pod replicas in all clusters.',
   CONTAINER_NAME: 'Container Name',
   CONTAINER_TYPE: 'Container Type',
   USE_DEFAULT_PORTS: 'Use Default Ports',
+  USE_IMAGE_DEFAULT_PORTS: 'Use Image Default Ports',
```
Comment (harrisonliu5) [inline locales/en/l10n-multiclusterProjects-applicationWorkloads-deployments-list.js:62]: only should review this word, thank @Bettygogo2021

Comment (Bettygogo2021) [inline locales/en/l10n-multiclusterProjects-applicationWorkloads-deployments-list.js:62]: Use Default Port of Images

**Review by Bettygogo2021 — CHANGES_REQUESTED** [review]

File: locales/en/l10n-multiclusterProjects-applicationWorkloads-deployments-list.js (hunk 1 above)
Comment (harrisonliu5) [reply locales/en/l10n-multiclusterProjects-applicationWorkloads-deployments-list.js]: done

**Pull request merged**